contracts/PositionManager.sol

```diff
@@ -135,6 +154,54 @@ contract PositionManager is IPositionManager, ERC721Holder {
         return uniswapNFTsMemory;
     }
 
+    ///@notice add awareness of aave position to positionManager
+    ///@param token address of token deposited
+    ///@param amount of aTokens recieved from deposit
+    function pushAavePosition(address token, uint256 amount) public {
+        StorageStruct storage Storage = PositionManagerStorage.getStorage();
+        uint256 shares;
+        //when i deposit into aave i get one share for each aToken recieved * shares_emitted/total_amount_already_recieved
```

Comment:
Typos and do not use the first person to refer to actual code execution in comments